### ISSUE TITLE
FHIR Code & Concept Conversion Ambiguity

### ISSUE THREAD

**Issue by bdrillard** [cqframework/clinical_quality_language#400]
[`InValueSet`](https://github.com/cqframework/clinical_quality_language/blob/master/Src/cql-lm/schema/elm/clinicalexpression.xsd#L327) is defined as checking code membership against a specified ValueSet. Although the function is only documented for `Code` types, it appears to have worked for `Concept` types as well in previous versions of the CQL-to-ELM project. Consider

```
library Query version '1'

using FHIR version '3.0.1'
include FHIRHelpers version '3.0.1' called FHIRHelpers

valueset "OurValueSet": '0.0.0.0.0.0.0.0.0.0'

context Patient

define TestInValueSet: [Observation] O where O.code in "OurValueSet"
```

which currently fails to compile with,

```
startLine: 10
startChar: 44
endLine: 10
endChar: 66
message: Ambiguous implicit conversion from FHIR.CodeableConcept to System.Concept or System.Concept.
errorType: semantic
errorSeverity: error

startLine: 10
startChar: 38
endLine: 10
endChar: 66
message: Expected an expression of type 'System.Boolean', but found an expression of type '<unknown>'.
errorType: semantic
errorSeverity: error
```

It appears the usage of a `FHIRHelpers.ToConcept` fails to be inferred. The code successfully compiles if the FHIRHelper function is applied directly, i.e.

```
define TestInValueSet: [Observation] O where FHIRHelpers.ToConcept(O.code) in "OurValueSet"
```

1. Should the documentation for `InValueSet` be amended to indicate the function is overloaded for the `Concept` type as well as `Code`? That might be its own sort of action item here.
2. Is this a new bug in implicit applications of FHIRHelpers functions?

**Comment by brynrhodes** [comment]
Thank you for the detailed report! This issue is caused by an incorrect additional implicit conversion declaration in the 3.0.1 model info. I've addressed this in the same push for #399.

As far as the documentation question, yes, that should definitely be clarified, and I will submit a ballot comment on 1.4 to correct it.

**Comment by brynrhodes** [comment]
Adressed in [1.3.14](https://github.com/cqframework/clinical_quality_language/releases/tag/v1.3.14)
